docs/changelog/91813.yaml

```
@@ -0,0 +1,5 @@
+pr: 91813
+summary: Skip remote clusters when performing up front privileges validation
+area: Transform
```

I changed the release note to specifically say datafeeds because DFA doesn't currently support CCS.

```suggestion
summary: Skip remote clusters when performing up front privileges validation for datafeeds
area: Machine Learning
```

---

I'll apply this suggestion once the release note file re-appears.

---

Done.